### PR TITLE
[FLN-2018] Adding date, location and new organizers.

### PR DESCRIPTION
**Pull request by perylemke**
Hi everybody,

Now we have a date, location and new organizers to DoD Florianópolis 2018 :)

Best regards,

Pery.

### CODE REVIEW

File: data/events/2018-florianopolis.yml [hunk 1]
```diff
@@ -56,13 +56,32 @@ team_members: # Name is the only required field for team members.
...
+    linkedin: "https://www.linkedin.com/in/isaacfsouza/"
+  - name: "Christopher Giese"
+    email: "christopher.giese@gmail.com"
+    employer: "Softplan"
+    linkedin: "https://www.linkedin.com/in/christopher-giese-kblo-aba41917/"
+  - name: "Thiago Trennepohl"
+    email: "thiagotrennepohl@gmail.com"
+    employer: "Involves"
+    linkedin: "https://www.linkedin.com/in/thiago-trennepohl-a897b396/"
 organizer_email: "organizers-floripa-2018@devopsdays.org" # Put your organizer email address here
```
Comment (phrawzty) [inline data/events/2018-florianopolis.yml]: These email addresses don't exist. I'll set up the list now based on the contents of `team_members:`.

cc @gomex

File: data/events/2018-florianopolis.yml [hunk 2]
```diff
@@ -72,7 +91,7 @@ sponsors:
   - id: samplesponsorname
     level: gold
  #  url: http://mysponsor.com/?campaign=me # Use this if you need to over-ride a sponsor URL.   
-  - id: arresteddevops
+  - id: sudocast
```
Comment (MikeRosTX) [inline data/events/2018-florianopolis.yml]: **_yml_** and **_logo_** missing for **_sudocast_**.

Comment (phrawzty) [reply data/events/2018-florianopolis.yml]: Oops, I didn't see this. 😞  @perylemke, you'll want to set up a PR to address this asap.

Comment (perylemke) [reply data/events/2018-florianopolis.yml]: Hey guys,

https://github.com/devopsdays/devopsdays-web/pull/4669

It's correct?

**Pull request merged**